src/audio/module_adapter/CMakeLists.txt

```diff
@@ -1,81 +1,71 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
-if(NOT CONFIG_LIBRARY)
-	if (CONFIG_IPC_MAJOR_4 OR CONFIG_CADENCE_CODEC OR (NOT CONFIG_COMP_LEGACY_INTERFACE) OR CONFIG_COMP_FIR)
-	add_local_sources(sof module_adapter.c module/generic.c)
-	endif()
-
-	if(CONFIG_COMP_VOLUME)
-	add_local_sources(sof
-		module/volume/volume_generic.c
-		module/volume/volume_hifi3.c
-		module/volume/volume_generic_with_peakvol.c
-		module/volume/volume_hifi3_with_peakvol.c
-		module/volume/volume.c)
-	endif()
-
-	if(CONFIG_CADENCE_CODEC)
-	add_local_sources(sof module/cadence.c)
-
-	if(CONFIG_CADENCE_CODEC_WRAPPER)
-	sof_add_static_library(codec_wrapper_lib ${CONFIG_CADENCE_CODEC_WRAPPER_LIB})
-	endif()
-
-	if(CONFIG_CADENCE_CODEC_AAC_DEC)
-	sof_add_static_library(xa_aac_dec ${CONFIG_CADENCE_CODEC_AAC_DEC_LIB})
-	endif()
-
-	if(CONFIG_CADENCE_CODEC_BSAC_DEC)
-	sof_add_static_library(xa_bsac_dec ${CONFIG_CADENCE_CODEC_BSAC_DEC_LIB})
-	endif()
-
+add_local_sources(sof module_adapter.c module/generic.c)
+
+if(CONFIG_COMP_VOLUME)
+  add_local_sources(sof
+    module/volume/volume_generic.c
+    module/volume/volume_hifi3.c
+    module/volume/volume_generic_with_peakvol.c
+    module/volume/volume_hifi3_with_peakvol.c
+    module/volume/volume.c)
+endif()
 
-	if(CONFIG_CADENCE_CODEC_DAB_DEC)
-	sof_add_static_library(xa_dab_dec ${CONFIG_CADENCE_CODEC_DAB_DEC_LIB})
-	endif()
+if(CONFIG_CADENCE_CODEC)
+  add_local_sources(sof module/cadence.c)
 
+  if(CONFIG_CADENCE_CODEC_WRAPPER)
+    sof_add_static_library(codec_wrapper_lib ${CONFIG_CADENCE_CODEC_WRAPPER_LIB})
+  endif()
 
-	if(CONFIG_CADENCE_CODEC_DRM_DEC)
-	sof_add_static_library(xa_drm_dec ${CONFIG_CADENCE_CODEC_DRM_DEC_LIB})
-	endif()
+  if(CONFIG_CADENCE_CODEC_AAC_DEC)
+    sof_add_static_library(xa_aac_dec ${CONFIG_CADENCE_CODEC_AAC_DEC_LIB})
+  endif()
```

Comment:
Avoid whitespace changes
https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes

Reply:
There's now a separate commit for indents. I just changed it to use tabs similarly as previously.